src/pages/login/login.js

```diff
@@ -133,6 +152,11 @@ const FormikSign = ({ classes, setAuthentication, setUser, props }) => (
         resetForm();
       } catch (error) {
         console.log(error);
+
+        enqueueSnackbar("Error on login, contact an admin", {
```

Comment:
@Sphinxs This message should me more clear to the user.
"contact an admin" sounds like an internal error, which is not true.
It's just the password (or e-mail) that are incorrect.

Reply:
Yep, I have removed the contact admin into another PR